NOTICE

```diff
@@ -0,0 +1,5 @@
+autowarefoundation/autoware-documentation
+Copyright 2021 The Autoware Foundation
+
+This product includes software developed at
```

Comment:
```suggestion
This product includes software developed by
```

Reply:
@xmfcx Could you please tell me the reason? It seems both `at` and `by` are used seeing some files.

https://github.com/apache/arrow-rs/blob/729934c5311b42fc7fd3522a22ddc049bdde9569/NOTICE.txt#L69-L70

https://github.com/apache/arrow-rs/blob/729934c5311b42fc7fd3522a22ddc049bdde9569/NOTICE.txt#L80-L84

Reply:
`by` is more major? :thinking: 
https://github.com/apache/solr/blob/main/NOTICE.txt

Reply:
I guess it depends on how we view a company.
- As a single unit -> `by` is appropriate.
- As an organization -> `at` can be used.

Right?

Reply:
At is for referencing to a place. By is for referencing it as a subject. AWF is less tied to a place than that company probably.

Reply:
Oh, this was for The AWF. (I thought it was for TIER IV.)

@xmfcx In that case, I'd like to follow The Apache Software Foundation.
I feel we can say it as a place because it's a community.

https://github.com/apache/arrow-rs/blob/729934c5311b42fc7fd3522a22ddc049bdde9569/NOTICE.txt#L4

https://github.com/apache/solr/blob/a63a6fb3091d8b21da453c9caf712882d2664e94/NOTICE.txt#L6-L7

Reply:
`at` here looks weird. https://github.com/apache/arrow-rs/blob/729934c5311b42fc7fd3522a22ddc049bdde9569/NOTICE.txt#L80 It says by the apache foundation too. The code is getting developed by the foundation.

Reply:
https://github.com/silentsoft/oss-notice-file-generator this tool is made to generate notice files and it puts `by`

Reply:
Hmm, for each contributor, it's fine.

But we follow Apache-2.0 and they use `at`, so I believe `at` is better for The AWF.
https://www.apache.org/legal/src-headers.html#notice

Reply:
@LalithVipulananthan @BonoloAWF @mitsudome-r Could you give me some comments about this? :pray:

Reply:
`at` attributes physical location. We are distributed around the world. https://english.stackexchange.com/questions/122929/developed-by-vs-developed-at#:~:text=When%20you%20say%20developed%20by%20ABC%2C%20inc.,of%20the%20company%20in%20question.

The code is developed by the AWF. not at.

Reply:
In that case, why does The Apache Foundation use `at`? :thinking: 

In my understanding,
- A foundation is a community, it's a place.
- Most code is not developed by the foundation itself, but in the community.

For example, TIER IV has contributed a lot of code to the AWF, but I feel it's unnatural to say it as "it was developed by the AWF".
"It was developed (by TIER IV) at the AWF" would be correct.

Reply:
https://docs.oracle.com/cloud/latest/big-data-discovery-cloud/BDDLG/cl_apache_commons_math_1_2.htm#BDDLG-concept_76CC91F563674CD7BF3BBFBDD8999555C9

Reply:
I guess some similar discussions are happened in the past and changed from `by` to `at`.
https://github.com/apache/httpd/commit/74a6ac31d18599f0dd58e0f443ca6843471c465b